src/elements/select/readme.md

```diff
@@ -70,6 +70,18 @@ the selected values will be displayed in selection order, separated by a comma.
 </sbb-form-field>
 ```
 
+## Style
+
+The component has no `size` property but, when slotted in a `sbb-form-field`,
+the height of the panel's trigger adapts based on the parent `size` property.
```

Comment:
```suggestion
it adapts to the parent `size` property.
```
Too technically deep imho

Reply:
Maybe "The `sbb-select` adapts its size to the `sbb-form-field`" would be enough